src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -16,7 +16,18 @@ namespace System.Net.Http.Tests
     public class HttpClientHandlerTest
     {
         readonly ITestOutputHelper _output;
+        private const string Username = "testuser";
```

Comment:
These are not respecting System.Net's test requirements: all constants regarding servers must be separated from the tests.